test/CalculatorGTest.cpp

```diff
@@ -0,0 +1,35 @@
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   CalculatorGtest.cpp
+ *  @date   Wed May 26 2021
+ *  @brief  Calculator Unit Tests
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include "../inc/Calculator.h"
+
+using namespace testing;
+
+//note formatCalculation does no calculation, it simply puts the numbers and char into a string
+TEST(formatCalculationTest, BasicFormat)
```

Comment:
Per Dune standards, the naming convention of unit test are, - When$FunctionGetsCalled_Then$SomethingHappenAndReturn where $FunctionGetsCalled is the name of the function you are invoking and $SomethingHappenAndReturn is the action and return type. An example is WhenGetImplementationIsCalledWithValidAccessPointDataWithNoIntentProfile_ThenApplicationAccessPointResponseReturnedWithoutIntentProfile